extensions/cornerstone/src/utils/mpr/toggleMPRHangingProtocol.ts

```diff
@@ -0,0 +1,285 @@
+import { Enums } from '@cornerstonejs/tools';
```

Comment:
Thank you for splitting this out - it makes it easier to understand to have things in smaller pieces.